diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/BlankLinesRule.kt

```diff
@@ -44,6 +46,16 @@ class BlankLinesRule(configRules: List<RulesConfig>) : DiktatRule(
             it.elementType == BLOCK && it.treeParent?.elementType != SCRIPT ||
                     it.elementType == CLASS_BODY || it.elementType == FUNCTION_LITERAL
         }) {
+            node.findParentNodeWithSpecificType(LAMBDA_ARGUMENT)?.let {
+                if ((node.treeNext?.treeNext?.elementType == RBRACE) && (node.treePrev.elementType == LBRACE)) {
```

Comment:
What does `treeNext.treeNext` here correspond to? If `node` is `WHITE_SPACE` here, I'd expect `treeNext` to be a `LBRACE`.

Reply:
Lambda body is always has a BLOCK
{ } - (LBRACE, WHITE_SPACE, BLOCK "", RBRACE)

Reply:
Oh, that's not obvious :) Please add a comment in code describing it.